src/js/containers/Home.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+// Imported Components
+import Post from '../components/Post'
+
+// Imported Actions
+import * as actions from '../actions/index'
+
+const mapStateToProps = (state) => {
+  return {
+    posts: state.posts
+  }
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return bindActionCreators(actions, dispatch);
+};
+
+
+class Home extends React.Component {
+  render() {
+    return (
+      <div>
+        <div className="col-xs-10 col-xs-offset-1 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3">
+          {this.props.posts.map((post, i) => <Post key={i} i={i} post={post} />)}
```

Comment:
IMO I would move this out to outside the JSX embedded into a `const`.  I've seen this style though and think it's popular too.  I also don't like the prop of `i`.  `index` or `postIndex` might be more clear.